package.json

```diff
@@ -6,7 +6,7 @@
     "displayName": "Azure Databases",
```

Comment:
In other repos, I had to add this to get the build to pass when updating the azure packages:
https://github.com/microsoft/vscode-azurefunctions/blob/main/tsconfig.json#L23

May or may not help the build error you're seeing for this PR

Reply:
It works, thanks! I was a `'async' modifier cannot be used in an ambient context` error in the `vscode-azureextensionui` package.